tembo-cli/tembo/Dockerfile.template

```diff
@@ -42,13 +42,31 @@ ENV PGDATABASE=postgres
 ENV PGUSER=postgres
 
 {% for trunk_install in trunk_installs %}
+{% if trunk_install.name == "pgvector" %}
 {% if trunk_install.version %}
+{% set VECTOR_VER = trunk_install.version %}
+{% else %}
+{% set VECTOR_VER = "0.6.0" %}
+{% endif %}
+# Install vector from source, this extension uses
+# chip-native optimizations, so binary distributions
+# are not compatible on Docker, when running on a Mac.
+USER root
+RUN mkdir /tmp/workdir && \
+    cd /tmp/workdir && \
+    git clone https://github.com/pgvector/pgvector.git && \
+    cd pgvector && \
+    git fetch origin v{{VECTOR_VER}} && \
+    git checkout v{{VECTOR_VER}} && \
+    make && make install && \
+    rm -rf /tmp/workdir
+USER postgres
+{% elif trunk_install.version %}
```

Comment:
this is working, but for some reason docker keeps rebuilding on every apply, which seems pretty inconvenient